File: CS2313/Week 4/Lecture 8/GradeBookApp/GradeBookApp/GradeBookApp.cpp

#include "GradeBook.h"

int main() {

	string courseName{ "Object Oriented Programming I" };
	array<int, GradeBook::numOfStudents> grades{ 20, 35, 70, 80, 90, 74, 62, 100, 66, 54 };

	GradeBook myGradeBook{courseName, grades};
	myGradeBook.processGrades();
	return 0;
}

File: CS2323/Chapter 7/spring22-chapter7/src/GradeBookApp.java
public class GradeBookApp {
    
    public static void main(String[] args) {
        double[] grades = {100, 89, 78, 10, 55};
        GradeBook cs2323 = new GradeBook(grades);

        System.out.println(cs2323.calcAverage());
    }
}
